components/post_view/embedded_forms/action_menu/action_menu.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {ActionResult} from 'mattermost-redux/types/actions';
+
+import {Post} from 'mattermost-redux/types/posts';
+
+import {AppBinding, AppCall} from 'mattermost-redux/types/apps';
+
+import {AppBindingLocations, AppExpandLevels} from 'mattermost-redux/constants/apps';
+
+import MenuActionProvider from 'components/suggestion/menu_action_provider';
+import AutocompleteSelector from 'components/autocomplete_selector';
+import PostContext from 'components/post_view/post_context';
+
+type Option = {
+    text: string;
+    value: string;
+};
+
+type Props = {
+    post: Post;
+    binding: AppBinding;
+    userId: string;
+    actions: {
+        doAppCall: (call: AppCall) => Promise<ActionResult>;
+    };
+};
+
+type State = {
+    selected?: Option;
+};
+
+export default class ActionMenu extends React.PureComponent<Props, State> {
+    private providers: MenuActionProvider[];
+
+    constructor(props: Props) {
+        super(props);
+
+        const binding = props.binding;
+        this.providers = [];
+        if (binding.bindings) {
+            const options = binding.bindings.map((b) => {
+                return {text: b.label, value: b.location};
+            });
+            this.providers = [new MenuActionProvider(options)];
+        }
+
+        this.state = {};
```

Comment:
any default selected value?

Reply:
Not for P1. Right now bindings don't have that functionality, and we must consider where to put it.